src/Tools/Source/RunTests/Program.cs

```diff
@@ -107,6 +147,76 @@ private static void WriteLogFile(Options options)
         }
 
         /// <summary>
+        /// Invoked when a timeout occurs and we need to dump all of the test processes and shut down 
+        /// the runnner.
+        /// </summary>
+        private static void HandleTimeout(Options options)
+        {
+            string GetProcDumpFilePath(Process proc) => ProcessUtil.Is64Bit(proc)
+                ? Path.Combine(options.ProcDumpPath, "procdump64.exe")
+                : Path.Combine(options.ProcDumpPath, "procdump.exe");
```

Comment:
Mostly paranoia.  The correct dumping behavior across process architectures can only be implemented by spawning a capture process with the desired architecture (at least according to the limited documentation I can find).  

There is no documentation on how this process spawning works.  Assuming it just looks for the correct process next to it.  But I'm unsure if it correctly passes along the `--accepteula` flag.  If it didn't it would result in a Jenkins hang.  Seems wrong to fix a jenkins hang with a jenkins hang.  So just maknig it simple here and picking the right architecture.

Would be willing to undo this if we could find some docs that point to `--acepteula` being passed along correctly.  Can't test locally though because I've already accepted it.